CHANGES.rst

```diff
@@ -8,11 +8,21 @@
 Changes
 =======
 
+Version 1.4.1 (released TBD)
+
+- set Sphinx ``<3`` because of errors related to application context
```

Comment:
**comment**: clear example that we need documentation guidelines. This repo is using lowercase, not dotted enumerations, while others (e.g. invenio-admin) totally the opposite. Not pointing fingers, just emphasising the need of what we learnt yesterday.

Reply:
This is following previous release changes digests. I would say that if we want to change we need to:
- Write somewhere how this should be done
- Use that format for the future
- Decide how to handle past occurrences of changelogs with an inconsistent format